.github/workflows/docker-test.yml

```diff
@@ -17,6 +17,10 @@ jobs:
 
     runs-on: ubuntu-latest
 
+    env:
+      DOCKER_SOLR_IMAGE_REPO: github-pr/solr
```

Comment:
Does this mean we will publish all of our PR docker images somewhere?

Reply:
nah this doesn't actually push. It's just testing that the custom repo/tag works.

I'm pretty sure it would fail if it tried, because github-pr isn't a docker hub repo.